# Семинар 4

1. Случайная непрерывная величина A имеет равномерное распределение на промежутке $(200, 800]$. Найдите ее среднее значение и дисперсию. Найдите ее среднее значение и дисперсию.

Используем формулы для определения мат. ожидания и дисперсии для случайной непрерывной величины $X$, равномерно распределённой на отрезке $[a;b]$:     "$\displaystyle\text {Математическое ожидание: }M(X)=\frac{a+b}{2}\\\\\displaystyle\text {Дисперсия: }D(X)=\frac{(b-a)^2}{12}$  
    Вычислим:

In [ ]:
def mean_and_variance(a,b):
    return f'На промежутке ({a};{b}]\nСреднее значение: М(А) = {(a+b)/2: .2f}\nДисперсия: D(A) = {((b-a)**2)/12: .2f}'
print(mean_and_variance(200,800))

На промежутке (200;800]
Среднее значение: М(А) =  500.00
Дисперсия: D(A) =  30000.00


Ответ:    $M(A)=500.00\\D(A)=30000.00$

2. О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2. Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5? Если да, найдите ее.

Зная, что дисперсия случайной непрерывной равномерно распределённой на отрезке $(a;b)$величины $B$ находится по формуле: $\displaystyle D(B)=\frac{(b-a)^2}{12}$  
И имея заданные значения $D(B)=0.2$ и $a=0.5$  
Найдем правую границу $b$ величины $B$ из уравнения
$\displaystyle \frac{(b-0.5)^2}{12}=0.2\\b-0.5=\sqrt{2.4}\\b=0.5+\sqrt{2.4}$  И найдем среднее значение по формуле:
$\displaystyle M(B)=\frac{a+b}{2}$

In [ ]:
b=0.5+2.4**(1/2)
print(f'Левая граница распределения величины В - b = {b: .3f}\n'
      f'Среднее значение В на промежутке (0.5; {b: .3f}) M(B) = {(b+0.5)/2: .3f}'
     )

Левая граница распределения величины В - b =  2.049
Среднее значение В на промежутке (0.5;  2.049) M(B) =  1.275


Ответ:

Левая граница распределения: $b=2.049$

Среднее значение: $M(B)=1.275$



3. Непрерывная случайная величина X распределена нормально и задана плотностью распределения $\displaystyle f(x)=\frac{1}{4\cdot\sqrt{2\pi}}\cdot e^{-\frac{(x+2)^2}{32}}$ , Найдите:  
а) $M(X)$  
б) $D(X)$  
в) $\displaystyle\sigma$ (среднее квадратичное отклонение)  

Из общей формулы плотности нормального распределения:  
$\displaystyle f(x)=\frac{1}{\sigma\sqrt{2\pi}}\cdot e^{-\frac{(x-a)^2}{2\sigma^2}}$  
$M(X)=a=-2\sigma = 4D(X)=\sigma^2=16$


Ответ:  
$M(X)=-2D(X)=16\displaystyle\sigma=4$



4. Рост взрослого населения города X имеет нормальное распределение. Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см. Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:  
а) больше 182 см  
б) больше 190 см  
в) от 166 см до 190 см  
г) от 166 см до 182 см  
д) от 158 см до 190 см
е) не выше 150 см или не ниже 190 см  
ё) не выше 150 см или не ниже 198 см  
ж) ниже 166 см.


Напишем функцию для определения $Z$-значений для данного распределения:

In [ ]:
def z_value(height):
    return (height-174)/8

Для расчета значений вероятности воспользуемся методом cdf, объекта NormalDist стандартного модуля statistics

In [ ]:
from statistics import NormalDist



a) Определим $Z$
-значениe:

In [ ]:
print(z_value(182))

1.0


метод cdf определяет вероятность того, что случайная величина X будет меньше или равна x, т.е в данном случае $P(X\le182)$

In [ ]:
NormalDist().cdf(z_value(182))

0.8413447460685429

Однако нам нужна вероятность $P(X\ge182)$, поэтому вычтем полученное значение из 1:

In [ ]:
P=1-NormalDist().cdf(z_value(182))
P

0.15865525393145707

$P(X\ge182)=0.1587$

б) Определим $Z$-значениe:

In [ ]:
z=z_value(190)
z

2.0



Определим $P(X\ge190)$

In [ ]:
P=1-NormalDist().cdf(z)
P

0.02275013194817921

$P(X\ge190) = 0.02275$

в) Определим $Z_1$
и $Z_2$

In [ ]:
z1=z_value(166)
z1

-1.0

In [ ]:
z2=z_value(190)
z2

2.0

Определим $P(166\le X\le 190)$

In [ ]:
P=NormalDist().cdf(z2)-NormalDist().cdf(z1)
P

0.8185946141203637

$P(166\le X\le 190) = 0.8186$

г) Определим $Z_1$
и $Z_2$

In [ ]:
z1=z_value(166)
z1

-1.0

In [ ]:
z2=z_value(182)
z2

1.0

Определим $P(166\le X\le 182)$

In [ ]:
P=NormalDist().cdf(z2)-NormalDist().cdf(z1)
P

0.6826894921370859

$P(166\le X\le 182) = 0.6827$

д) Определим $Z_1$
и $Z_2$

In [ ]:
z1=z_value(158)
z1

-2.0

In [ ]:
z2=z_value(190)
z2

2.0

Определим $P(158\le X\le 190)$

In [ ]:
P=NormalDist().cdf(z2)-NormalDist().cdf(z1)
P

0.9544997361036416

$P(158\le X\le 190) = 0.9545$

е) Определим $Z_1$
и $Z_2$

In [ ]:
z1=z_value(150)
z1

-3.0

In [ ]:
z2=z_value(190)
z2

2.0

Определим $P(X\le150\cup X\ge 190)$

In [ ]:
P=NormalDist().cdf(z1)+(1-NormalDist().cdf(z2))
P

0.024100029979809312

$P(X\le150\cup X\ge 190) = 0.0241$

ё) Определим $Z_1$ и $Z_2$

In [ ]:
z1=z_value(150)
z1

-3.0

In [ ]:
z2=z_value(198)
z2

3.0

Определим $P(X\le150\cup X\ge 198)$

In [ ]:
P=NormalDist().cdf(z1)+(1-NormalDist().cdf(z2))
P

0.002699796063260207

$P(X\le150\cup X\ge 198) = 0.0027$

ж) Определим $Z$-значениe:

In [ ]:
z=z_value(166)
z

-1.0

Определим $P(X\le166)$

In [27]:
P=NormalDist().cdf(z)
P

0.15865525393145707

$P(X\le166) = 0.1587$

Ответ:  
а) $P(X\ge182)=0.1587$;  
б) $P(X\ge190) = 0.02275$;  
в) $P(166\le X\le 190) = 0.8186$;  
г) $P(166\le X\le 182) = 0.6827$;  
д) $P(158\le X\le 190) = 0.9545$;  
е) $P(X\le150\cup X\ge 190) = 0.0241$;  
ё) $P(X\le150\cup X\ge 198) = 0.0027$;  
ж) $P(X\le166) = 0.1587$

5. На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?

Воспользуемся формулой расчета $Z$
-оценки: $\displaystyle Z=\frac{X-\bar x}{\sigma}=\frac{190-178}{\sqrt{25}}=\frac{12}{5}$  

In [28]:
Z=(190-178)/25**(1/2)
Z

2.4

In [29]:
print(f'Рост человека, равный 190 см, отклоняется от математического ожидания роста в популяции, \n'
      f'в которой M(X) = 178 см и D(X) = 25 кв.см на Z = {Z} сигм.')

Рост человека, равный 190 см, отклоняется от математического ожидания роста в популяции, 
в которой M(X) = 178 см и D(X) = 25 кв.см на Z = 2.4 сигм.


Ответ:

на $2.4$
сигмы